### PR TITLE
Add code to review

### PR DESCRIPTION
**Pull request by GraceHtet**
Added html, css and js
Added webpack 
Added linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+- **[WEBPACK]**
+  <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (MarkoKermi) [inline README.md:107]: As the project looks nice and it's in final stage, it's nice to also have the live version in the `readme` file 🔥

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+      listFuns.edit(text.value, idx);
+    };
+
+    text.addEventListener('keydown', (e) => {
+      if (e.key === 'Enter' || e.key === 'Tab') textnormal(text);
+    });
+  });
+
+  localStorage.setItem('tdlists', JSON.stringify(listFuns.tdlists));
+};
```
Comment (MarkoKermi) [inline src/index.js:68]: Optional!
As far as I can see you have the `edit` todo part also in `localstorage`, which part of the requirements ( all changes are supposed to be stored in local storage), but in the live version I can't see that functionality. Maybe we can check that together, se we can both confirm that is implemented!

**Review by MarkoKermi — CHANGES_REQUESTED** [review]

**Review by MarkoKermi — CHANGES_REQUESTED** [review]
Hello @GraceHtet, 
The project looks very nice, great job 👏

- No linters errors 👍 
- Nice design  ✔️ 
- Correct Github flow ✔️

Required Changes ♻️

You really did a great job but there are some features you need to improve

- [ ]  Please check the comments below
I'm sure you can do it just keep it up

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: README.md (hunk 1 above)
Comment (MarkoKermi) [inline README.md:107]: As the project looks nice and it's in final stage, it's nice to also have the live version in the readme file.

File: src/index.js (hunk 2 above)
Comment (MarkoKermi) [inline src/index.js:68]: - [ ] It's clear that you add the `edit` function in `localstorage` as  it's part of the requirements (all changes are supposed to be stored in local storage), but the function it's not visible in live version. Maybe we can check that together just to be sure that is implemented!

**Review by MarkoKermi — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (GraceHtet) [reply src/index.js:68]: Thank You  .I've fixed it.

**Pull request merged**